### DISCUSSION TITLE
Develope this software for Mac OS

### DISCUSSION THREAD

**Discussion by pablogh1** [amir1376/ab-download-manager#45] [Ideas]
Hi
PC has IDM and i think it doesn't need such software. Instead Mac OS doesn't have any propriate internet download manager. I think you should spent your time to develope this software for mac.
Thanks.

**Comment by amir1376** [comment]
Hi there. Unfortunately I don't have macOS device to test the app. 
I see that someone compile it for macOS and it works properly but If I release it for macOS. I can't test or fix possible bugs. so for now I decided to not do that

**Reply by andymachs** [reply]
> Hi there. Unfortunately I don't have macOS device to test the app. I see that someone compile it for macOS and it works properly but If I release it for macOS. I can't test or fix possible bugs. so for now I decided to not do that

Hi, there, You can install virtual machine on Windows, and then install macOS on virtual machine.

**Reply by amir1376** [reply]
I don't like develop things this way. let alone developing for a vendor lock in system!

**Comment by andymachs** [comment]
If AB download manager for macOS is better than IDM for Windows, That is a great things in the world. because on macOS, there is not a good download tool. this is a good chance.

**Reply by AliYar-Khan** [reply]
there are no paid download manager as well for MacOS ?

**Reply by driscollxy** [reply]
Please consider using Free Download Manager (FDM) for MacOS while waiting a chance for AB Download Manager to become available. Although FDM does not integrate with Safari, it is compatible with popular web browsers such as Chrome, Edge, and Firefox.

**Comment by Alrazzouq** [comment]
Hi ... I hope to make a version for MacOS.

**Comment by amir1376** [comment]
The macOS version is available now! ( well at least for more than a year:) )

**Reply by LiveOnlyFans** [reply]
nice
